frontend/assets/stylesheets/components/_event-detail.scss

```diff
@@ -513,3 +514,18 @@
 .ticket-sales__item__date {
     float: right;
 }
+.ticket-sales__list {
+    @include fs-data(4);
+    color: $c-neutral2;
+}
+
+/* ==========================================================================
+   Google Map
+   ========================================================================== */
+.google-map {
```

Comment:
Do we need to style this directly, is there a more general way we can express these styles given it's just a font-size?